alert.go

```diff
@@ -780,8 +792,10 @@ func (a *AlertNode) renderMessageAndDetails(id, name string, t time.Time, group
 
 func (a *AlertNode) handlePost(post *pipeline.PostHandler, ad *AlertData) {
 	bodyBuffer := a.bufPool.Get().(*bytes.Buffer)
-	defer a.bufPool.Put(bodyBuffer)
-	bodyBuffer.Reset()
+	defer func() {
+		bodyBuffer.Reset()
+		a.bufPool.Put(bodyBuffer)
```

Comment:
OK, sounds good. I don't think its a big deal either, if it comes up later we can update.